### ISSUE TITLE
redundancy reducer

### ISSUE THREAD

**Issue by mattanimation** [exodrifter/unity-rumor#27]
this is more of a feature request and it might be totally out of the question, but I've noticed some  redundancy when scripting where for example:

```
choice "I have a belly button."
    say "I have a belly button."
```
the choice and the say are the same, what if I could do this instead

```
choice "Would you tell me more?"
    say "~"
```

where the ~ was interpreted by the parser to replace it with the parent choice text

just a thought, it could be whatever symbol or I guess if it were a variable or something that could be used for both the choice and say option, but anyhoo.
this message will self-destruct in...

**Comment by exodrifter** [comment]
This is very similar to #24, which will add a way to get the contents of the last chosen choice. This is an interesting idea too, and I feel like it fixes some gripes I had in issue 24.

**Comment by exodrifter** [comment]
Merged issue into #24.
